t/90dtrace.t

```diff
@@ -51,10 +51,16 @@ if ($tracer_pid == 0) {
         or die "failed to create temporary file:$tempdir/trace.out:$!";
     if ($^O eq 'linux') {
         exec qw(bpftrace -v -B none -p), $server->{pid}, "-e", <<'EOT';
+struct st_h2o_http2_frame_t {uint32_t length; uint8_t type; uint8_t flags; uint32_t stream_id;};
```

Comment:
I can see this being necessary for completeness. But it comes with redundancy (complexity).

As stated above, I am tempted to suggest with just starting with the frame type. We can change things when it turns out that we need more information.